unqualified name

```sh
Error: In file included from ./src/stdgo/sync/atomic/_Atomic/Atomic_Fields_.cpp:14:
include/stdgo/sync/atomic/_Atomic/Atomic_Fields_.h:12:37: error: expected identifier or '{'
HX_DECLARE_CLASS4(stdgo,sync,atomic,_Atomic,Atomic_Fields_)
                                    ^
include/stdgo/sync/atomic/_Atomic/Atomic_Fields_.h:12:1: error: expected unqualified-id
HX_DECLARE_CLASS4(stdgo,sync,atomic,_Atomic,Atomic_Fields_)
^
/usr/local/lib/haxe/lib/hxcpp/4,3,2/include/hx/Macros.h:21:66: note: expanded from macro 'HX_DECLARE_CLASS4'
#define HX_DECLARE_CLASS4(ns4,ns3,ns2,ns1,klass) namespace ns4 { HX_DECLARE_CLASS3(ns3,ns2,ns1,klass) }
                                                                 ^
/usr/local/lib/haxe/lib/hxcpp/4,3,2/include/hx/Macros.h:20:62: note: expanded from macro 'HX_DECLARE_CLASS3'
#define HX_DECLARE_CLASS3(ns3,ns2,ns1,klass) namespace ns3 { HX_DECLARE_CLASS2(ns2,ns1,klass) }
                                                             ^
/usr/local/lib/haxe/lib/hxcpp/4,3,2/include/hx/Macros.h:19:58: note: expanded from macro 'HX_DECLARE_CLASS2'
#define HX_DECLARE_CLASS2(ns2,ns1,klass) namespace ns2 { HX_DECLARE_CLASS1(ns1,klass) }
                                                         ^
/usr/local/lib/haxe/lib/hxcpp/4,3,2/include/hx/Macros.h:18:52: note: expanded from macro 'HX_DECLARE_CLASS1'
#define HX_DECLARE_CLASS1(ns1,klass) namespace ns1 { HX_DECLARE_CLASS0(klass) }
                                                   ^
In file included from ./src/stdgo/sync/atomic/_Atomic/Atomic_Fields_.cpp:14:
include/stdgo/sync/atomic/_Atomic/Atomic_Fields_.h:17:11: error: expected identifier or '{'
namespace _Atomic{
          ^
include/stdgo/sync/atomic/_Atomic/Atomic_Fields_.h:17:18: error: expected unqualified-id
namespace _Atomic{
                 ^
./src/stdgo/sync/atomic/_Atomic/Atomic_Fields_.cpp:54:11: error: expected identifier or '{'
namespace _Atomic{
          ^
./src/stdgo/sync/atomic/_Atomic/Atomic_Fields_.cpp:54:18: error: expected unqualified-id
namespace _Atomic{
                 ^
6 errors generated.
```

hxcpp: 4.3.2
haxe: 5.0.0-alpha.1+f25d42c

[Repo to reproduce](https://github.com/PXshadow/unqualifiedname_hxcpp)

--- a/src/generators/gencpp.ml
+++ b/src/generators/gencpp.ml
@@ -393,7 +393,12 @@ let keyword_remap name =
    | "HX_" | "HXLINE" | "HXDLIN"
    | "NO" | "YES"
    | "abstract" | "decltype" | "finally" | "nullptr" | "static_assert"
-   | "struct" -> "_hx_" ^ name
+   | "struct" | "_Atomic"
+   | "constexpr" | "consteval" | "constinit"
+   | "co_await" | "co_return" | "co_yield"
+   | "alignas" | "alignof"
+   | "_Alignas" | "_Alignof"
+   | "requires" -> "_hx_" ^ name
    | x -> x
 ;;
 

